### DISCUSSION TITLE
Max zoom, locked area and swipe on TabBar

### DISCUSSION THREAD

**Discussion by robyf70** [fleaflet/flutter_map#1127] [Q&A]
Can you please suggest how can I lock a map into a particular lat/long at max zoom and be able to swipe to a next tab in TabBar without being able to move the map?

**Comment by robyf70** [comment]
I've just found that I can use 
```dart
zoom: 18.0,
minZoom: 18.0,
maxZoom: 18.0,
interactiveFlags: InteractiveFlag.none,
```
but how to propagate the fingermove to the TabBar?

**Reply by mziadd** [reply]
+1

**Reply by robyf70** [reply]
Just to report that I've currently workaround the problem by putting the Map into a Stack with a transparent Container on top. In this way the swipe are captured by the transparent Container and the map stay locked and the TabBar swipes left and right.
